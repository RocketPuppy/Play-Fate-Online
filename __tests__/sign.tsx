import * as React from "react";
import * as renderer from "react-test-renderer";
import Sign from "../src/sign";

test("sign renders a plus when number is positive", () => {
  const positiveSign = renderer.create(<Sign num={1} />);

  expect(positiveSign.toJSON()).toMatchSnapshot();
});

test("sign renders a minus when number is negative", () => {
  const negativeSign = renderer.create(<Sign num={-1} />);

  expect(negativeSign.toJSON()).toMatchSnapshot();
});
