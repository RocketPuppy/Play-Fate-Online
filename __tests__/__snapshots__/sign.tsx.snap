// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sign renders a minus when number is negative 1`] = `
<span>
  -
</span>
`;

exports[`sign renders a plus when number is positive 1`] = `
<span>
  +
</span>
`;

exports[`sign renders nothing when number is 0 1`] = `<span />`;
